### PR TITLE
Touchpad/Trackpad scrolling IE10+ and Edge

### PR DESCRIPTION
**Pull request by TheBoef**
Right now you can't scroll with your touchpad/trackpad on a element that
has the perfect scrollbar in IE10+ and Edge.
By using the property '-ms-overflow-style' (Supported in IE10+ and Edge)
too hide the overflow instead of the default 'overflow' property, you
keep the scrolling with touchpad/trackpad functionality.

### CODE REVIEW

**Comment by hatashiro** [comment]
Thank you very much! Can anybody please test this out?

**Comment by DanielApt** [comment]
I can test on my IE10/11 VM. 

Don't have Edge.

**Comment by ghost** [comment]
This appears to have fixed our scrolling issues with IE11 on Surface tablets. Thanks!

**Comment by iver56** [comment]
@noraesae This change didn't make it into 0.6.9. Try to run `bower install perfect-scrollbar` and search for `touch-action:` (which was one of the things this pull request added) in the downloaded code.

TL;DR: Touch scrolling still doesn't work in Edge in 0.6.9

**Comment by iver56** [comment]
It seems that this commit is the culprit:
https://github.com/noraesae/perfect-scrollbar/commit/3e76ce9d3ed291941f600f9121a2f59b857235a3

**Comment by hatashiro** [comment]
True! Sorry for the mistake. Will fix now.

**Updated: Fixed and released as 0.6.10. Thanks!**

**Comment by iver56** [comment]
Splendid! Now it works

**Comment by DanielApt** [comment]
:+1:
